rmw_fastrtps_shared_cpp/src/rmw_take.cpp

```diff
@@ -175,11 +175,15 @@ __rmw_take(
   bool * taken,
   rmw_subscription_allocation_t * allocation)
 {
-  RCUTILS_CHECK_FOR_NULL_WITH_MSG(
-    subscription, "subscription pointer is null", return RMW_RET_ERROR);
-  RCUTILS_CHECK_FOR_NULL_WITH_MSG(
-    ros_message, "ros_message pointer is null", return RMW_RET_ERROR);
-  RCUTILS_CHECK_FOR_NULL_WITH_MSG(taken, "boolean flag for taken is null", return RMW_RET_ERROR);
+  RMW_CHECK_FOR_NULL_WITH_MSG(
+    subscription, "subscription handle is null",
+    return RMW_RET_INVALID_ARGUMENT);
+  RMW_CHECK_FOR_NULL_WITH_MSG(
+    ros_message, "ros message handle is null",
+    return RMW_RET_INVALID_ARGUMENT);
```

Comment:
@Lobotuerk nit: `ros_message` is no handle. I believe you want to use [`RMW_CHECK_ARGUMENT_FOR_NULL`](https://github.com/ros2/rmw/blob/efe3f2bf932aabd70e875228e39b843103f2c4b3/rmw/include/rmw/error_handling.h#L107). Same elsewhere.

Reply:
Updated for RMW_CHECK_ARGUMENT_FOR_NULL